package.json

```diff
@@ -18,5 +18,8 @@
   "bugs": {
     "url": "https://github.com/Equal-Vote/star-server/issues"
   },
-  "homepage": "https://github.com/Equal-Vote/star-server#readme"
+  "homepage": "https://github.com/Equal-Vote/star-server#readme",
+  "dependencies": {
+    "crypto": "^1.0.1"
```

Comment:
Is this being used?

Reply:
it's being used in IRequest.ts
not sure how it was working before without the import?  (could have been a transient dependency anyway??)